src/internal/FluencePeer.ts

```diff
@@ -94,20 +102,37 @@ interface ConnectionInfo {
  */
 export class FluencePeer {
     /**
-     * Creates a new Fluence Peer instance. Does not start the workflows.
-     * In order to work with the Peer it has to be initialized with the `init` method
+     * Creates a new Fluence Peer instance.
      */
     constructor() {}
 
     /**
-     * Get the information about Fluence Peer connections
+     * Checks whether the object is instance of FluencePeer class
+     * @param obj - object to check if it is FluencePeer
+     * @returns true if the object is FluencePeer false otherwise
+     */
+    static isInstance(obj: FluencePeer): boolean {
+        if (obj && obj._isFluenceAwesome) {
```

Comment:
`(obj isInstanceOf FluencePeer) || (obj && obj._isFluenceAwesome)`?

Reply:
`(obj && obj._isFluenceAwesome)`  might return undefined. I wanted the return type to be strictly true\false

Reply:
Yeah, I'm about to check `isInstanceOf` on the first try.

Reply:
The compiler isn't happy:

`property '_isFluenceAwesome' does not exist on type 'never'.ts(2339)`